css/style.css

```diff
@@ -1,19 +1,358 @@
+  margin: 100px auto 20px;
+  text-align: center;
+  color: var(--clr-black);
+
+  padding-bottom: 80px;
+}
+
+.second-section-container h2 {
+  font-weight: 300;
+  font-size: clamp(22px, 3.8vw, 4vw);
+  margin-bottom: 1.6em;
+}
+.article-container {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: space-around;
+  width: 80vw;
+  margin: 0 auto;
+}
+
+.article-container img {
+  width: 12vw;
+  margin-bottom: 34px;
+}
+
+.article-container p {
+  font-size: 1.5rem;
+}
+/* =======feature section ======== */
+.feature-section {
+  display: flex;
+  background-color: var(--clr-orange-light);
+  margin: 20px 0 50px;
+}
+
+.feature-section img {
+  width: 40vw;
+}
+
+.feature-section-wrapper {
+  padding: 8em 10vw;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.feature-section-wrapper h3 {
+  font-size: 1.9rem;
+  font-weight: 400;
+  margin: 0 0 1.2em;
+  text-align: center;
+}
+
+.feature-section-wrapper h3::before {
+  content: "“";
+  font-size: 2.4rem;
+  color: var(--clr-orange);
+}
+
+.feature-section-wrapper h3::after {
+  content: "”";
+  font-size: 2.4rem;
+  color: var(--clr-orange);
+}
+
+.feature-section-wrapper a {
+  text-decoration: none;
+}
+/* ============Footer============= */
+
+footer {
+  margin: 0 auto;
+  width: 88%;
+  border-top: #d4d4d4 solid 1px;
+  text-align: center;
+  font-size: 0.9rem;
+  padding: 20px 0 10px;
 }
 
-/**
- * Add your custom styles below
- * 
- * Remember: 
- * - Be organised, use comments and separate your styles into meaningful chunks
- *    for example: General styles, Navigation styles, Hero styles, Footer etc.
- * 
- * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
- */
+footer p {
+  color: var(--clr-black);
+}
+
+footer img {
+  width: 2rem;
+  margin: 12px 10px;
+  border: var(--clr-light-gray) 1px solid;
+  border-radius: 50%;
+  padding: 10px;
+}
+
+footer p:last-child {
+  color: var(--clr-light-gray);
+  margin-bottom: 40px;
+}
+
+/* ///////////////////////// store page //////////////////////////// */
+
+/* ============== main ============== */
+
+.store-page-main {
+  display: flex;
+}
+.store-page-main img {
+  width: 50%;
+  height: 100%;
+  background-color: white;
+}
+
+fieldset {
+  border: none;
+}
+
+.form-container {
+  width: 50vw;
+  padding: 5em 5em 0 5em;
+}
 
+.form-container h1 {
+  font-size: 2.3rem;
+  color: var(--clr-orange);
+  font-weight: 500;
+  margin-bottom: 1.8em;
+}
+
+label,
+form p {
+  color: rgb(91, 91, 91);
+  font-weight: 500;
+  margin: 20px 0 10px;
+}
+
+input,
+select {
+  border: 2px solid rgb(91, 91, 91);
+  border-radius: 6px;
+  padding: 10px 0px 10px 10px;
+  font-size: 1.2rem;
+  color: rgb(91, 91, 91);
+}
+
+form div {
+  display: flex;
+  flex-direction: column;
+}
 
+.small-input {
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.small-input div {
+  width: 47.5%;
```

Comment:
Instead of setting a fixed % width here, you could make these divs use `flex: 1` (expand to fill available space) and either:

- add a right margin to the first div
- OR add a `gap` property to the parent to add the desired fixed space between them (just like you did [here](https://github.com/CodeYourFuture/HTML-CSS-Module-Project/pull/440/files#diff-1fc556f95754ee7e33d91044125c44bb9f750c99be4406756ffb27413adfcaf5R43))